insert_by_period/macros/get_period_sql.sql

```diff
@@ -5,9 +5,9 @@
 {% macro default__get_period_sql(target_cols_csv, sql, timestamp_field, period, start_timestamp, stop_timestamp, offset) -%}
 
   {%- set period_filter -%}
-    ("{{timestamp_field}}" >  '{{start_timestamp}}'::timestamp + interval '{{offset}} {{period}}' and
-     "{{timestamp_field}}" <= '{{start_timestamp}}'::timestamp + interval '{{offset}} {{period}}' + interval '1 {{period}}' and
-     "{{timestamp_field}}" <  '{{stop_timestamp}}'::timestamp)
+    ({{timestamp_field}} >  '{{start_timestamp}}'::timestamp + interval '{{offset}} {{period}}' and
+     {{timestamp_field}} <= '{{start_timestamp}}'::timestamp + interval '{{offset}} {{period}}' + interval '1 {{period}}' and
+     {{timestamp_field}} <  '{{stop_timestamp}}'::timestamp)
```

Comment:
Does the `timestamp_field` not need to be quoted anymore?

Reply:
Not if we want to allow things like `timestamp_field = 'cast(created_at as timestamp)'` like I do in the integration test now

This offers more flexibility I feel (even if this is a change), and people could quote their own field in the config if they wanted:
```timestamp_field = '`my col with spaces`'```

Reply:
Makes sense 👍

It does make the name `timestamp_field` less accurate now. It's more like a "timestamp expression" now since it can either be a column name _or_ a SQL expression that yields a timestamp. But for backwards compatibility, we can leave it with the less-than-ideal original name.